templates/wordpress_vhost.j2

```diff
@@ -16,6 +16,9 @@
         <IfModule mod_php7.c>
```

Comment:
Don't really need to keep php7 support anymore, can just change all references to the version that works with php8.